init.ps1

```diff
@@ -57,7 +61,17 @@ function AppendContent($Message, $Uri, $BuildScriptPath, [switch]$Enable, [hasht
     if($enable){
         Write $Message
         $progressPreference = 'silentlyContinue'
-        $content = (Invoke-WebRequest -Uri $Uri -UseBasicParsing -Headers @{"Cache-Control"="no-cache"} ).content
+		$requestResult = (Invoke-WebRequest -Uri $Uri -UseBasicParsing -Headers @{"Cache-Control"="no-cache"} )

```

Comment:
I made this change so I could run the file locally.
When I executed this:
Invoke-WebRequest "file:///C:/Users/Justin%20Mason/Documents/repos/psakezero/init.ps1"  -UseBasicParsing | Invoke-Expression

And was trying to add my own components the $requestResult was a String.  When I did it against a HTTP uri it was an object with a content property.